data/reusables/repositories/custom-anchors.md

```diff
@@ -0,0 +1,25 @@
+Use HTML anchor tags (`<a id="unique-anchor-name"></a>`) to create navigation anchor points to any place in the document.
+
+> [!NOTE]
+> Custom anchors will not be included in the document outline/Table of Contents.
+
+Link to custom anchors by using the `id` given to the anchor.
+
+For example:
+
+```markdown
+# Section Heading
+
+Some body text of this section.
+
+<a id="my-custom-anchor-point"></a>
```

Comment:
Replacing with `name` to align with the information on contributing to GitHub docs.

```suggestion
<a name="my-custom-anchor-point"></a>
```